source/shared/cpp/ObjectModel/ParseUtil.h

```diff
@@ -5,6 +5,7 @@
 #include "pch.h"
 #include "AdaptiveCardParseException.h"
 #include "ParseContext.h"
+//#include "BaseCardElement.h"
```

Comment:
remove this

Reply:
thanks

---
In reply to: [282259135](https://github.com/Microsoft/AdaptiveCards/pull/2864#discussion_r282259135) [](ancestors = 282259135)